admin_apikey.proto

```diff
@@ -47,6 +47,8 @@ message ApiKey {
   optional int64 ts_end = 6;
   // Status of the ApiKey.
   ApiKeyStatus status = 7;
+  // An id of the key that is not sensitive.
+  uint32 public_id = 8;
```

Comment:
@nikron are you okay making this part of the public api? i don't see a reason why not, but just FYI, this is going public.